python/ray/tests/test_runtime_env_2.py

```diff
@@ -62,23 +62,15 @@ def f(self):
     ):
         ray.get(a.f.remote())
 
-    # The second time this runs it should be faster as the error is cached.
-    start = time.time()
     with pytest.raises(RuntimeEnvSetupError, match="ResolvePackageNotFound"):
         ray.get(f.options(runtime_env=bad_env).remote())
 
-    assert (time.time() - start) < (first_time / 2.0)
```

Comment:
do not quite understand with this needs to be `first_time / 2`

Reply:
I think it was just a guess for something in between the uncached speed and the cached speed. @SongGuyang might have more context about this part.